tests/phpunit/CRM/Utils/RuleTest.php

```diff
@@ -343,4 +344,38 @@ public function testCvvRule($cvv, $type, $expected): void {
     $this->assertEquals($expected, CRM_Utils_Rule::cvv($cvv, $type));
   }
 
+  /**
+   * Test CVV rule
+   *
+   * @param string $email
+   * @param bool $expected expected outcome of the rule validation
+   *
+   * @dataProvider emails
+   */
+  public function testEmailRule(string $email, bool $expected): void {
+    $this->assertEquals($expected, CRM_Utils_Rule::email($email));
+  }
+
+  /**
+   * Test that 'test@localhost' works in dev environments.
+   */
+  public function testDevelopmentEmail(): void {
+    Civi::settings()->set('environment', 'Development');
```

Comment:
@eileenmcnaughton This line needs to go too?

Reply:
@mattwire yeah - I just removed that test as it is meaningless now